tests/utils/fixtures/setups/ansible/setup.py

```diff
@@ -6,11 +6,12 @@
 import re
 import sys
 import warnings
-
 from collections import defaultdict
 from distutils.command.build_scripts import build_scripts as BuildScripts
 from distutils.command.sdist import sdist as SDist
 
+from ansible.release import __author__, __version__
```

Comment:
Why are these test files modified?

Reply:
They never prevented the tests to pass so I am not sure why they would need to be modified

Reply:
Good catch.

It is apparently result of applying isort and black on the code and in this case it is possibly wrong (as the import shall probably happen after path manipulation).

Simply, this change shall be rejected.

Reply:
Well, sorry if I was not clear but what I meant is that the fixtures `setup.py` files do not need to be modified as far as I know.

Reply:
The only modifications of `setup.py` currently are explicitly opening the file (often JSON one) using utf-8. If this is removed and given code runs (for whatever reason) on a system with other than utf-8 default encoding, there is a chance it would fail.

In `.../ansible/setup.py` are three such cases:
- dump JSON file
- load JSON file
- read requirements file

Regarding JSON files the rules are simple: open it in text mode and specify encoding utf-8. Dealing with JSON files differently is error prone to local default encoding.

Regarding reading requirements.txt - it seems to me safer to be explicit and do not allow any space for encoding related issues.

Do you want me to remove any of these three modifications?

Reply:
The fixtures files are taken directly from existing projects to help testing them as-is and, as such, should not be modified.

Reply:
Good. All changes to `...../ansible/setup.py` reverted.

Reply:
Thanks! However I also meant the other fixture files that you modified and not only the `ansible` one.

Reply:
Thanks for your endless patience.

setup.py for fixtures (3 cases) reverted to their original content.